scripts/htmlproofer-external

```diff
@@ -18,5 +18,8 @@ HTMLProofer.check_directory(directory, {
   },
   # treat absolute URLs to our domain as if they were local
   # helps for rel="canonical" tags that exist locally but are not deployed yet
-  url_swap: { %r|https://www.login.gov/| => '/' },
+  url_swap: {
+    %r|https?://(www\.)?login\.gov/?| => '/',
```

Comment:
if www is optional, maybe we should be more consistent which domain we use?

But a task for another time

Reply:
> if www is optional, maybe we should be more consistent which domain we use?

Yeah, it's a good point. I think I was most worried about keeping them consistent into the future, but maybe we could have an automated test to keep us consistent 🤔

Reply:
Another reason it'd be nice to enforce this: Right now, a lot of our links hit a redirect, which isn't great.

```bash
$ curl -I -s https://login.gov/what-is-login | grep location:
location: /what-is-login/
```

Reply:
Seems like two different cleanup tasks to me?
1. ensure local links always have trailing slash
2. pick a domain (either www or plain) to standardize on.

Re: domain, do we know which one we want? Personally I like the simplicity of "plain"

Reply:
> Re: domain, do we know which one we want? Personally I like the simplicity of "plain"

Interestingly, I discovered that the [config value](https://github.com/18F/identity-site/blob/8582ebf7938cdc555e15caab3744d2eb27cb67a1/_config.yml#L118) is `https://www.login.gov`, but then this is overridden to `https://login.gov` via Federalist settings.

But then separately there's a [`canonical_domain` setting](https://github.com/18F/identity-site/blob/8582ebf7938cdc555e15caab3744d2eb27cb67a1/_config.yml#L4) which is `www.login.gov`, and isn't overridden 😬  This is used for `<link rel="canonical">` meta tags.

I agree we should pick one and settle on it. Outside the canonical tag, the plain URL seems most common, and would be my preference as well.